src/main/java/com/teragrep/cfe_39/Config.java

```diff
@@ -0,0 +1,183 @@
+/*
+   HDFS Data Ingestion for PTH_06 use CFE-39
+   Copyright (C) 2022  Fail-Safe IT Solutions Oy
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
wrong license

Reply:
Solved in commit 8721d828a5a4f2630949a6c8225e6be472ac536b